dandiapi/api/views/asset.py

```diff
@@ -1,3 +1,16 @@
+try:
+    from storages.backends.s3boto3 import S3Boto3Storage
+except ImportError:
+    # This should only be used for type interrogation, never instantiation
+    S3Boto3Storage = type('FakeS3Boto3Storage', (), {})
+try:
+    from minio_storage.storage import MinioStorage
+except ImportError:
+    # This should only be used for type interrogation, never instantiation
+    MinioStorage = type('FakeMinioStorage', (), {})
```

Comment:
Is the point of this dual try-catch to always import either one or the other?

Reply:
I believe the idea is that in testing or local environments, `MinioStorage` will be available, and boto otherwise, but we need this code path to work in both situations.

If we could just have both that would make the code simpler, but maybe there's good reason to avoid that. @dchiquito will know details here.

Reply:
This code traces its lineage back to S3FF. I believe the original idea was that if an app was only going to use Minio, it should only need to pip install the minio package, not boto3, and vice versa. In DANDI I see that we are just requiring both all the time, but it's a nice thought.